app/controllers/books_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to book_url(@book), notice: t('controllers.books_controller.created') }
+        format.html { redirect_to book_url(@book), notice: t('controllers.controller.created', name: Book.model_name.human) }
```

Comment:
```suggestion
        format.html { redirect_to book_url(@book), notice: t('controllers.common.created', name: Book.model_name.human) }
```

特に命名規則があるわけでは無いのですが、controllers controllerは「馬から落ちて落馬して感」があるので、commonみたいな名前にするといいかもしれません

Reply:
たしかに違和感があります…今後は気をつけて命名します🙇🏻‍♀️